tests/lib/srxlo_test.py

```diff
@@ -81,6 +81,7 @@
     'dscp_set',
     'ether_type',
     'expiration',
+    'filter_term',
```

Comment:
Does the SRX support hierarchical filters? 

I can't quickly find documentation either way.

Reply:
This is called Nested filters and it doesn't seem like SRX supports this. 

https://apps.juniper.net/feature-explorer/feature-info.html?fKey=10519&fn=Nested%20Filters

Reply:
The changes otherwise look great! Thanks!

Reply:
If my understanding is correct, the `srxlo` generator is for firewall filters on SRX devices, which is different from the security policies of the `srx` target.
I tested the following on a SRX220 and it accepted the commit:
```
[edit]
+  firewall {
+      family inet {
+          filter test {
+              term foo {
+                  filter bar;
+              }
+          }
+          filter bar {
+              term accept {
+                  then accept;
+              }
+          }
+      }
+  }
```
If those assumptions are correct, I think it's fully supported. And different from nested filters, which only seem to be present on high end PTX (with no mention of MXs where I tested them as well).

Reply:
Yup, srxlo seems to be the loopback on SRX devices. Interesting, thanks for testing this out!